### PR TITLE
Add the ValoraPunks mainnet contract

### PR DESCRIPTION
**Pull request by silasbw**

### CODE REVIEW

**Comment by MuckT** [comment]
[Contract Address](https://explorer.celo.org/mainnet/address/0xaF99AC3aB1c17728d0Ee3256f02061018845b588)

File: src/mainnet-nft-list.json [hunk 1]
```diff
@@ -42,5 +42,9 @@
   {
     "name": "CyberBoxCertNFT",
     "contractAddress": "0xa4a8e345e1a88efc9164014bb2cebd4c2f98e986"
+  },
+  {
+    "name": "ValoraPunks",
+    "contractAddress": "0xaF99AC3aB1c17728d0Ee3256f02061018845b588"
```
Comment (cajubelt) [inline src/mainnet-nft-list.json:48]: this has some upper case characters, not sure if that's ok

Comment (cajubelt) [reply src/mainnet-nft-list.json:48]: if it's not we should add test coverage for that

**Review by MuckT — APPROVED** [review]
💯

File: src/mainnet-nft-list.json (hunk 1 above)
Comment (cajubelt) [reply src/mainnet-nft-list.json:48]: I added a test for address upper casing to valora-rest-api that can be imitated https://github.com/valora-inc/valora-rest-api/pull/228/files

Comment (cajubelt) [reply src/mainnet-nft-list.json:48]: oh looks like other addresses in this file are mixed case too

Comment (silasbw) [reply src/mainnet-nft-list.json:48]: It's ok.  We already have an annoying lack of checksummed and lowercased contract addresses.  I've meant to add a CI job to force one or the other.

**Pull request merged**